Content.Shared/Atmos/Components/MovedByPressureComponent.cs

```diff
@@ -27,5 +27,8 @@ public sealed partial class MovedByPressureComponent : Component
 
     [ViewVariables(VVAccess.ReadWrite)]
     public int LastHighPressureMovementAirCycle { get; set; } = 0;
+
+    [DataField]
+    public HashSet<string> TableLayerRemoved = new();
```

Comment:
Let's add a comment to remind us what this is for.

Reply:
Something along the lines of "The table layer gets removed by space wind so that... however we need to remember to put it back..."